independent-projects/tools/base-codestarts/src/main/filtered/codestarts/quarkus/tooling/gradle-wrapper/codestart.yml

```diff
@@ -1,4 +1,4 @@
-name: gradle-wrapper
+name: tooling-gradle-wrapper
```

Comment:
Maybe `core` or `quarkus-core` would be a better prefix?

But I don't mind if it stays that way. Some tests need adjustments though.

Reply:
I would prefer to keep tooling-...